.github/workflows/deploy-pages.yml

```diff
@@ -6,12 +6,19 @@ on:
 jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
+    
+    strategy:
+      matrix:
+        compiler: [gcc] # could put [gcc, clang] to see both work
+
     steps:
       - name: Checkout 🛎️
         uses: actions/checkout@v2.3.1
         with:
           persist-credentials: false
-
+        env:
+          CC:  ${{ matrix.compiler }}
```

Comment:
This is a surprising location for this variable - I expect it later, when "wasm-pack build" is called.

You could test these changed by temporarily removing restriction to main at the start of the file.

Reply:
Just following those links I provided. Jonathan doesn't think it'll make a difference anyway. So, i'll close this.